app/presenters/wpcc/message_presenter.rb

```diff
@@ -40,5 +40,18 @@ def above_max_contribution
 
       t('wpcc.contributions.contribution_gt40000_warning', amount: amount)
     end
+
+    def employee_contribution_tip
```

Comment:
We may need a presenter spec in `spec/presenters/message_presenter_spec.rb` to the methods `employer_contribution_tip` and `employee_contribution_tip`. There's an example for a similar method, `manually_opt_in_message?` within the same file.

Reply:
👍  tests added